javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,71 @@
+var width;
+var height;
+var area;
+
+width = 5;
+height = 6;
+area = width * height;
```

Comment:
This is not necessary in JavaScript, you can assign a value on the same line you declare a variable 🤓

Reply:
just fixed, thanks for the heads up!

Reply:
<img width="1278" alt="Screen Shot 2022-05-16 at 3 35 56 PM" src="https://user-images.githubusercontent.com/103273265/168693722-4749b263-58dc-4563-b5b4-fe356c93ee81.png">